documentation/examples/prometheus.yml

```diff
@@ -9,10 +9,17 @@ global:
   external_labels:
       monitor: 'codelab-monitor'
 
+# List of Alertmanager instances
```

Comment:
The comment "List of Alertmanager instances" isn't 100% accurate for the `alerting` section, as it can also define other settings, discover via SD, etc.

How about just "Alertmanager configuration."?

Reply:
Indeed ; fixed